string color
float64 distance_m
float64 world_x
float64 world_y
string image_path
